tests/test_split_corpus.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
+
+import pytest
+from fixtures import DataDir
+
+from pipeline.common.downloads import read_lines
+
+# With 47 lines, there will be 5 lines per file, except the last file which should have 2.
+corpus_line_count = 47
+
+
+@pytest.mark.parametrize("task", ["src-en", "trg-ru"])
+def test_split_mono(task: str):
```

Comment:
It's probably still useful but FYI there's also more high level https://github.com/mozilla/firefox-translations-training/blob/main/tests/test_split_collect.py